src/main/java/org/folio/circulation/domain/FeeFineOwner.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.circulation.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineOwner {
+  private final JsonObject representation;
+
+  public FeeFineOwner() {
+    representation = null;
+  }
+
+  public FeeFineOwner(JsonObject representation) {
+    this.representation = representation;
+  }
```

Comment:
Similar to for Fee / Fine, please replace this with fields for the state instead of JSON

Reply:
Done.